data/env/snapd.fish.in

```diff
@@ -1,12 +1,14 @@
 # Expand $PATH to include the directory where snappy applications go.
-set -u snap_bin_path "@SNAP_MOUNT_DIR@/bin"
-fish_add_path -aP $snap_bin_path
+set -ul snap_bin_path "@SNAP_MOUNT_DIR@/bin"
+if not contains $snap_xdg_path $PATH
```

Comment:
Shouldn't this be:
```suggestion
if not contains $snap_bin_path $PATH
```
? If not, I think I'm missing the point

Reply:
Thanks for spotting. I'll force push. It must have been was then injected by systemd environment generator then when I tested that